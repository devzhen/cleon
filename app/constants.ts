export const DEFAULT_COORDS = {
  top: 100,
  left: 220,
} as const;

export const BOARD_TYPE = {
  left: 'left',
  right: 'right',
} as const;