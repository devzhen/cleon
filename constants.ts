export const DEFAULT_STICKER_POS = {
  top: 100,
  left: 220,
} as const;

export const BOARD_TYPE = {
  left: 'left',
  right: 'right',
  all: 'all',
} as const;

export const STICKERS_INITIAL_VALUE = { [BOARD_TYPE.left]: {}, [BOARD_TYPE.right]: {}};
