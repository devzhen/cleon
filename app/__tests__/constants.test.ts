import * as constants from '../constants';

describe('constants', () => {
  it('matches snapshot', () => {
    expect(constants).toMatchSnapshot();
  });
});
