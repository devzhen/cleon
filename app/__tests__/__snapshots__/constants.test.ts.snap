// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants matches snapshot 1`] = `
{
  "BOARD_TYPE": {
    "all": "all",
    "left": "left",
    "right": "right",
  },
  "DEFAULT_STICKER_POS": {
    "left": 220,
    "top": 100,
  },
  "STICKERS_INITIAL_VALUE": {
    "left": {},
    "right": {},
  },
}
`;
